projects/1.16/assets/resourceful-bees/resourcefulbees/lang/zh_cn.json

```
@@ -1,442 +1,827 @@
+	"item.resourcefulbees.scraper.tooltip.1": "必须先收集完所有的蜜脾, 才能让蜜蜂产出新的蜜脾！",
+	"block.resourcefulbees.apiary_breeder.tooltip.info": "这是一个可选的培育器, 它可以被添加进多方块结构养蜂场的一部分。",
+	"block.resourcefulbees.apiary_breeder.tooltip.info1": "最大培育时间: ",
+	"block.resourcefulbees.apiary_breeder.tooltip.info2": "可与额外的培育器升级或培育时间升级一同升级。",
+	"item.resourcefulbees.apiary_breed_time_upgrade.tooltip.info": "最大培育时间减少300刻。",
+	"item.resourcefulbees.apiary_breeder_upgrade.tooltip.info": "增加额外的培育器升级到蜂场培育器",
+	"block.resourcefulbees.centrifuge.tooltip.structure_size": "多方块结构离心机尺寸是 3x4x3, 由一个控制器和其余的外壳组成。",
+	"block.resourcefulbees.centrifuge.tooltip.requisites": "控制器必须高于底部一格并且在任何面的中间。",
+	"block.resourcefulbees.centrifuge.tooltip.capabilities": "多方块结构里的任意方块都可以进行物品和能量的输入与输出。",
+	"block.resourcefulbees.beehive.tooltip.bees": "蜜蜂: ",
+	"block.resourcefulbees.beehive.tooltip.honeycombs": "蜜脾: ",
+	"item.resourcefulbees.hive_upgrade.tooltip.info": "在前一等级的蜂巢上使用以升级它",
+	"block.resourcefulbees.honey_glass.tooltip": "允许蜜蜂穿过。",
+	"block.resourcefulbees.honey_glass_player.tooltip": "允许玩家通过。",
+
+	"effect.resourcefulbees.calming": "镇定",
+	"item.minecraft.potion.effect.calming": "镇定药水",
+	"item.minecraft.potion.effect.long_calming": "镇定药水",
+	"item.minecraft.splash_potion.effect.calming": "喷溅型镇定药水",
+	"item.minecraft.splash_potion.effect.long_calming": "喷溅型镇定药水",
+	"item.minecraft.lingering_potion.effect.calming": "滞留型镇定药水",
+	"item.minecraft.lingering_potion.effect.long_calming": "滞留型镇定药水",
+	"item.minecraft.tipped_arrow.effect.calming": "镇定之箭",
+	"item.minecraft.tipped_arrow.effect.long_calming": "镇定之箭",
+
+
+
+	"book.resourcefulbees.name": "五十度蜂",
+	"book.resourcefulbees.landing_text": "欢迎缤纷养蜂人！本指南中蜜蜂的情色内容要比起养蜂人的信息要少得多。我们希望它能帮助你更好地理解基础知识。$(br)$(o)$(#ff0000)请记住大多数 (不是全部) 的蜜蜂都是由开发者添加的！$()$(br )中文翻译: $(br)Holy_Fak (在版本0.5.8b时翻译) $(br)Nuogz (在版本0.6.6a时翻译) ",
+
+	"book.resourcefulbees.getting_started": "入门",
+	"book.resourcefulbees.getting_started.desc": "本节介绍如何开始游玩这个模组，例如如何寻找和捕获蜜蜂、如何安全地存储它们以及如何跟踪你遇到的蜜蜂。",
+
+	"book.resourcefulbees.getting_started.finding_bees": "寻找蜜蜂",
+	"book.resourcefulbees.getting_started.finding_bees.desc": "环游世界。$(br2) 环游世界。",
+	"book.resourcefulbees.getting_started.finding_bees.text.1": "蜜蜂可以在蜂巢中找到，它们也可以在世界各地漫游，像其他生物一样繁殖。$(br2)如果你想知道蜜蜂可以在哪里产卵，你可以通过$(or)$(l:beekeeping_tools/beepedia)蜜蜂百科$()或在JEI中查找蜜蜂。",
+	"book.resourcefulbees.getting_started.finding_bees.text.2":  "有些蜜蜂不会自然繁殖，而是需要育种。$(thing)$(l:getting_started/breeding_bees)这里$()你可以找到有关繁殖蜜蜂的更多信息。",
+
+	"book.resourcefulbees.getting_started.collecting_bees": "收集蜜蜂",
+	"book.resourcefulbees.getting_started.collecting_bees.desc": "去把他们全都抓起来。",
+	"book.resourcefulbees.getting_started.collecting_bees.text": "一旦你找到了一些蜜蜂，想要收集它们并将它们带回家。$(br2)有几种方法可以做到这一点，最好的办法是使用 $(or)$(l:beekeeping_tools/bee_jar)蜂罐$() 或者$(or)$(l:beekeeping_tools/bee_box)蜂盒$().$(br2)，你也可以使用传统的方法比如栓绳，或蜜蜂育种相关物品进行操作。",
+
+	"book.resourcefulbees.getting_started.understanding_bees": "认识（了解）蜜蜂",
+	"book.resourcefulbees.getting_started.understanding_bees.desc": "知识百科.",

```

```suggestion
	"book.resourcefulbees.getting_started.understanding_bees": "认识蜜蜂",
	"book.resourcefulbees.getting_started.understanding_bees.desc": "蜜蜂知识百科。",
```
原文：Encyclobeedic knowledge.